Source/g_game.c

```diff
@@ -1540,7 +1540,8 @@ static void G_DoSaveGame(void)
   memset (name2,0,sizeof(name2));
 
   // killough 2/22/98: "proprietary" version string :-)
-  sprintf (name2,VERSIONID,MBFVERSION);
+  //sprintf (name2,VERSIONID,MBFVERSION);
+  sprintf (name2,VERSIONID,MBF21VERSION);
```

Comment:
After a quick look at the `save_g.c` file, I think compatibility with old saves is difficult. My first idea is rewriting most of it in the style of Chocolate Doom. Do you have ideas?

Reply:
That will be a huge amount of work, dissecting all the structs into their individual members. The advantage, though, would be that we could support both 32 and 64 bit saves and even old saves from MBF.exe.

Reply:
I just don't see any other way. Even I have saves and I don't want to break them :)

Reply:
> I just don't see any other way. Even I have saves and I don't want to break them :)

I see.

> Or port Woof to Choco/Crispy as another Doom engine. Maybe it's even easier.

Not in this case. The main advantage would be shared audio/video/input code - but I think we got this quite right already. The savegame code would have to be rewritten one way or the other.

Reply:
> The main advantage would be shared audio/video/input code

Also netcode, much better dehacked and opl emulation. Sometimes I seriously think about it.